### PR TITLE
LPS-202859  Test fix| master

### PR DESCRIPTION
**Pull request by leilanyaragao**
Jira Issue: https://liferay.atlassian.net/browse/LPS-202859

### CODE REVIEW

**Comment by leilanyaragao** [comment]
ci:test:sf

**Comment by leilanyaragao** [comment]
ci:test:sf

File: modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/OpenAPIResourceTest.java [hunk 1]
```diff
@@ -515,6 +511,7 @@ private JSONObject _toJSONObject(String yamlString) throws Exception {
 	private static final String _OBJECT_FIELD_NAME =
 		"x" + RandomTestUtil.randomString();
 
+	@DeleteAfterTestRun
 	private static Company _company;
```
Comment (seltong) [inline modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/OpenAPIResourceTest.java:515]: if we apply the previous suggestion, please remove the `@DeleteAfterTestRun`

**Review by seltong — CHANGES_REQUESTED** [review]

File: modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/OpenAPIResourceTest.java [hunk 2]
```diff
@@ -69,15 +68,12 @@ public class OpenAPIResourceTest {
 	public static final AggregateTestRule aggregateTestRule =
 		new LiferayIntegrationTestRule();
 
-	@BeforeClass
-	public static void setUpClass() throws Exception {
+	@Before
+	public void setUp() throws Exception {
```
[the remark this answers is not in the document]
Comment (leilanyaragao) [reply modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/OpenAPIResourceTest.java:72]: @seltong I tried this way, but it wasn't possible. I will talk to you on slack.

**Review by seltong — APPROVED** [review]
LGTM :tada:

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**